dev-python/pipenv/pipenv-2020.4.1_beta1.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python Development Workflow for Humans"
+HOMEPAGE="https://github.com/pypa/pipenv https://pypi.org/project/pipenv/"
+SRC_URI="https://github.com/pypa/pipenv/archive/v2020.4.1b1.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+PATCHES=(
+	"${FILESDIR}/disable-networked-tests.patch"
+	)
+
```

Comment:
There is `PATCHES` twice now. Please drop this copy.

Reply:
Oops, that was not supposed to be there twice.